### ISSUE TITLE
Feature Request: More detailed information in Monitoring on Events tab

### ISSUE THREAD

**Issue by waterlink** [trufflesuite/truffle-teams#7]
Hi there, thank you so much for creating Truffle Teams!

I’ve just started using this for monitoring of our smart contracts (trying it out on Ropsten), and the first thing I was hopeful to see is the list of events that happened in the contract. And while there is a list there, it doesn’t have much details.

Since I have uploaded the ABI of the contract (that includes the info about types and names of event’s fields), I expect to see them there.

I understand that this is beta, and the feature is maybe somewhere in your backlog down the line.

Could you tell me if this is something you plan to introduce or not?

**Comment by mikeseese** [comment]
Hey @waterlink! I apologize for the delay! Generally speaking: yes we want to add more contextual information for events in transactions (similar to [what you'd see on the Ganache GUI](https://www.trufflesuite.com/docs/ganache/truffle-projects/events-page)). This would include the event name (which should already be displaying; if you see `Unknown Event`, we should get into that), the event signature (parameters and their types), and the values of those parameters. Is this what you're thinking?

It's definitely in the backlog and something we really want to add (the TruffleCon 2018 tagline was "Less hex, more text" haha).

**Comment by waterlink** [comment]
@seesemichaelj 

> Unknown event

No, I’m getting appropriate names of the events (e.g. `Transfer`)

> This would include the event name ... the event signature ... and the values of those parameters. Is this what you're thinking?

That’s exactly what I was thinking!
